dace/transformation/interstate/gpu_transform_sdfg.py

```diff
@@ -312,11 +338,13 @@ def apply(self, _, sdfg: sd.SDFG):
             for node in state.nodes():
                 if sdict[node] is None:
                     if isinstance(node, (nodes.LibraryNode, nodes.NestedSDFG)):
-                        node.schedule = dtypes.ScheduleType.GPU_Default
-                        gpu_nodes.add((state, node))
+                        if node.guid not in self.host_maps and not self._output_or_input_is_marked_host(state, node):
```

Comment:
If the node is a LibraryNode or NestedSDFG, why would its ID be in `host_maps`? A documentation line clarifying that `host_maps` can also be used for the above SDFG elements suffices.

Reply:
I decided to limit the extension to only consider maps for now.